RetailCoder.VBE/Root/RubberduckModule.cs

```diff
@@ -306,6 +306,11 @@ private void BindQuickFixTypes(IEnumerable<Assembly> assemblies)
             foreach (var quickFix in quickFixes)
             {
                 Bind<IQuickFix>().To(quickFix).InSingletonScope();
+                if (quickFix.Name.Equals("AssignedByValParameterMakeLocalCopyQuickFix"))
```

Comment:
This is not refactoring-friendly: if/when `AssignedByValParameterMakeLocalCopyQuickFix` gets renamed, this code will break. Also, why does *that one* needs its own specific binding rule?